.github/workflows/ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         cache: 'pnpm'
     - run: pnpm install
     - run: pnpm run test:unit:coverage
-    - uses: codecov/codecov-action@v2.1.0
```

Comment:
Is due to broken rebase that we see that ?

Reply:
yes :)